src/Microsoft.Azure.ServiceBus/Amqp/AmqpMessageSession.cs

```diff
@@ -8,7 +8,6 @@ namespace Microsoft.Azure.ServiceBus.Amqp
     using System.Threading.Tasks;
     using Azure.Amqp;
     using Core;
-    using Messaging.Amqp;
 
     class AmqpMessageSession : MessageSession
```

Comment:
and rename it to MessageSession 

---
In reply to: [113602592](https://github.com/Azure/azure-service-bus-dotnet/pull/146#discussion_r113602592) [](ancestors = 113602592)